microservices-connector/config/HPA/customMetrics.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Could we add the logic to update manifest files for GMC in update_manifests.sh ?

Reply:
Maybe after #337 has been fixed?

Reply:
I can do it as a separate PR.